pom.xml

```diff
@@ -462,6 +461,7 @@
               <exclude>**/*.avsc</exclude>
               <exclude>**/*.avro</exclude>
               <exclude>**/*.log</exclude>
+              <exclude>**/R/**</exclude>
```

Comment:
@bzz The compiled R package gets installed under R/, so we do need an exclusion for that in the top level pom.  Its a different R directory from r/R (which is source).  

I've narrowed the exclusion and included a comment in what I'm about to push.